Stats/Supporting Files/es.lproj/Localizable.strings

```diff
@@ -7,7 +7,7 @@
 //  Running on macOS 10.15.
 //
 //  Copyright © 2020 Serhiy Mytrovtsiy. All rights reserved.
-//
+//f
```

Comment:
This change seems to be unintentional.
```suggestion
//
```

Reply:
Apologies for that, probably a failed `CMD+F` 😅